src/GuestConfiguration/GuestConfiguration/Az.GuestConfiguration.psd1

```diff
@@ -104,14 +104,10 @@ PrivateData = @{
         # IconUri = ''
 
         # ReleaseNotes of this module
-        ReleaseNotes = '* Initial release of Az.GuestConfiguration module. Provides these two cmdlets.
-  * Get-AzVMGuestPolicyReport
-    * Provides compliance status of a VM in a resource group, compliance reasons.
-  * Get-AzVMGuestPolicyStatusHistory:
-    * Provides historical compliance statuses of a VM in a resource group, for a maximum of past 14 days.'
+        ReleaseNotes = 'Initial Release'
```

Comment:
Comments from the changelog below should go here

Reply:
fixed